src/MetaMorpho.sol

```diff
@@ -737,6 +744,8 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
 
                 marketConfig.enabled = true;
 
+                _updateLastTotalAssets(lastTotalAssets + MORPHO.expectedSupplyAssets(marketParams, address(this)));
```

Comment:
If not documented yet, consider documenting the side effects of this operation. The share value (of MM) is increased.